moodle/ruleset.xml

```diff
@@ -110,6 +110,11 @@
     -->
     <rule ref="moodle.PHPUnit.TestCaseProvider"/>
 
+    <!--
+        Tests should have a return type. The default return type should be void.
+    -->
+    <rule ref="moodle.PHPUnit.TestReturnType"/>
```

Comment:
Fun one... I just realised... why do we need to add any "moodle.*" to the ruleset file? Only if there is some exception to their handling (say, disable, or lower from error to warning, or configure something...) they are needed.

It seems that we have missed that detail lately and there are 2-3 cases that can be safely removed.

Or am I missing anything?